flair/trainers/trainer.py

```diff
@@ -227,7 +230,10 @@ def train(
         if type(base_path) is str:
             base_path = Path(base_path)
 
-        log_handler = add_file_handler(log, base_path / "training.log")
+        if create_file_logs:
```

Comment:
This is problematic. If you set create_file_logs=False and create_loss_file=False and use a new folder, the execution crashes since the folder is never created. 

You need to add the following snippet above to make sure the folder is always created: 


```python
        # create folder if it doesn't exist
        if not os.path.exists(base_path):
            os.makedirs(base_path)
```

Reply:
I see, I didn't test it with non-existing folders, now it should work, thanks